.travis.yml

```diff
@@ -9,15 +7,25 @@ matrix:
   include:
     - node_js: "0.10"
       env: BROWSER=true
-env: BROWSER=false
 before_install:
   - npm install -g npm
-  - if [[ $BROWSER == true ]]; then npm install karma-phantomjs-launcher karma-firefox-launcher; fi
+  - npm install -g karma-cli
```

Comment:
I'm not sure that `karma` would be added to path like that seeing the commands aren't being run in `npm`s context. Either way, it doesn't really matter as its only being installed in travis sandbox